### ISSUE TITLE
Add a README to the repo

### ISSUE THREAD

**Issue by MuditJ** [DavidDeCoding/kilo-go#6]
I think adding a README with a gif showing the working or just a simple image would be nice, it would show that the application is currently functional and get more people trying it out.

**Comment by DavidDeCoding** [comment]
Yes, I agree; it lacks some of the bells and whistles. If you are looking to write up the README, go for it.

**Comment by frink** [comment]
Also needs a license if you are inviting contribution. Without a specified license open source is murky due to copyright law in many countries. Most seasoned coders will not touch it unless it has a license. I'd suggest, MIT, Apache 2.0, BSD3, BSD2, BSD1 or if you're very adventurous you can do like the ubiquitous Rob Landley and go BSD0. Just please don't GPL the thing....

**Comment by DavidDeCoding** [comment]
Sure I can add a license.
